api/host.go

```diff
@@ -181,12 +182,13 @@ type (
 	}
 
 	HostScan struct {
-		HostKey    types.PublicKey `json:"hostKey"`
-		PriceTable rhpv3.HostPriceTable
-		Settings   rhpv2.HostSettings
-		Subnets    []string
-		Success    bool
-		Timestamp  time.Time
+		HostKey           types.PublicKey `json:"hostKey"`
+		PriceTable        rhpv3.HostPriceTable
```

Comment:
Not all properties have a `json` tag. Same goes `HostPriceTableUpdate` in the same file.